client/src/components/NavBar/CustomButton.tsx

```diff
@@ -0,0 +1,25 @@
+import Button from '@material-ui/core/Button';
+import { Link } from 'react-router-dom';
+
+interface Props {
+  linkTo: string;
+  btnText: string;
+}
+
+const CustomButton = ({ linkTo, btnText }: Props): JSX.Element => {
+  return (
+    <Link to={linkTo}>
```

Comment:
You can combine Link and Button here :)

Reference: https://stackoverflow.com/questions/38187833/how-to-combine-reactjs-router-link-and-material-ui-components-like-a-button

Reply:
Good to know, thanks for this!